lib/resources/r4/provenance.yaml

```diff
@@ -81,12 +81,65 @@ fields:
         }
       ]
     }
-  note: |
-    <ul>
-      <li> Only one agent may be provided. </li>
-      <li> All `coding` must contain the `system` and `code` fields. </li>
-    </ul>
+  note: <ul><li> Only one agent may be provided. </li></ul>
```

Comment:
Here, as well as line 15/32 and below, there are different formats for single-string note fields, could we have those all be the same?

Reply:
https://github.com/cerner/fhir.cerner.com/pull/371/commits/9a3653d0ebe32919d77441d9d12d2f25b685dc85